Mathlib/MeasureTheory/Integral/IntervalAverage.lean

```diff
@@ -56,3 +57,57 @@ theorem intervalAverage_congr_codiscreteWithin {a b : ℝ} {f₁ f₂ : ℝ →
     ⨍ (x : ℝ) in a..b, f₁ x = ⨍ (x : ℝ) in a..b, f₂ x := by
   rw [interval_average_eq, intervalIntegral.integral_congr_codiscreteWithin hf,
     ← interval_average_eq]
+
+/-- The mean value theorem for integrals. -/
```

Comment:
Could you expand the docstring, explaining what the theorem says (there are many values of the mean value theorem for integrals).